packages/utils/src/crypto.ts

```diff
@@ -92,6 +114,6 @@ async function generate32BufferKey(): Promise<string> {
  * @returns a string of 8 random bytes
  */
 async function generate8randomBytes(): Promise<string> {
-  const random32Bytes = await CryptoWrapper.random32Bytes();
-  return random32Bytes.slice(0, 8).toString('hex');
+  const random32BytesHex = await random32Bytes();
```

Comment:
Please revert the name of this variable back to `random32Bytes`

Reply:
The variable random32Bytes caused a naming collision because random32Bytes() is now being called directly. I changed it to random32BytesHex to indicate that it holds the hexadecimal value of the random bytes returned by the function, this is to prevent naming collision.

Other variable names that we could use would be randomBytes, randomHex, randomHexBytes.

Let me know if you have any specific suggestions for the variable name.